libs/content/workspace/src/tab/mod.rs

```diff
@@ -78,6 +60,18 @@ impl std::fmt::Debug for TabContent {
     }
 }
 
+// used by task manager to offload serialization to a background thread
+impl Clone for TabContent {
+    fn clone(&self) -> Self {
+        match self {
+            TabContent::Markdown(editor) => TabContent::Markdown(editor.clone()),
+            TabContent::Svg(svg) => TabContent::Svg(svg.clone()),
+            TabContent::Image(_) => unimplemented!("images aren't cloneable"),
+            TabContent::Pdf(_) => unimplemented!("pdfs aren't cloneable"),
+        }
+    }
+}
```

Comment:
This seems more acceptable to me, but I would also be pretty surprised to find a `.clone()` crashing my program.